app/next-client-app/components/data-table/index.tsx

```diff
@@ -68,6 +68,14 @@ export function DataTable<TData, TValue>({
     },
   });
 
+  const handleRowClick = (id: string) => {
+    const location = window.location.pathname;
+    window.location.href = UrlBuider(
```

Comment:
Not for now, but can we please correct the spelling of `UrlBuider` here and in its definition above?